From b15636e98316362fc00b00512f986cea910f7bfd Mon Sep 17 00:00:00 2001
From: Riddhi Aggarwal <quic_riddagga@quicinc.com>
Date: Thu, 1 Jun 2023 15:46:07 +0530
Subject: [PATCH] arm64: dts: qcom: ipq9574: Adding build support for Alder DB
 RDPs

Adding build support for the following DB RDPs
ipq9574-db-al01-c1
ipq9574-db-al01-c2
ipq9574-db-al01-c3
ipq9574-db-al02-c1
ipq9574-db-al02-c2
ipq9574-db-al02-c3

Change-Id: I6d676b8b4e2fda50743ba612c2968a45ad82cee7
Signed-off-by: Riddhi Aggarwal <quic_riddagga@quicinc.com>
---
 arch/arm/boot/dts/Makefile        | 6 ++++++
 arch/arm64/boot/dts/qcom/Makefile | 6 ++++++
 2 files changed, 12 insertions(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index abbdc3adbc27..666fc26da2d6 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -1060,6 +1060,12 @@ dtb-$(CONFIG_ARCH_QCOM) += \
 	ipq9574-rdp459.dtb \
 	ipq9574-rdp467.dtb \
 	ipq9574-rdp469.dtb \
+	ipq9574-db-al01-c1.dtb \
+	ipq9574-db-al01-c2.dtb \
+	ipq9574-db-al01-c3.dtb \
+	ipq9574-db-al02-c1.dtb \
+	ipq9574-db-al02-c2.dtb \
+	ipq9574-db-al02-c3.dtb \
 	ipq5332-rdp441.dtb \
 	ipq5332-rdp442.dtb \
 	ipq5332-rdp446.dtb \
diff --git a/arch/arm64/boot/dts/qcom/Makefile b/arch/arm64/boot/dts/qcom/Makefile
index 770da39f991f..d77b57c0d9ef 100644
--- a/arch/arm64/boot/dts/qcom/Makefile
+++ b/arch/arm64/boot/dts/qcom/Makefile
@@ -24,6 +24,12 @@ dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp458.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp459.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp467.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp469.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-db-al01-c1.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-db-al01-c2.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-db-al01-c3.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-db-al02-c1.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-db-al02-c2.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-db-al02-c3.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= msm8916-alcatel-idol347.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= msm8916-asus-z00l.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= msm8916-huawei-g7.dtb
-- 
2.34.1

